Source/Core/Core/Movie.cpp

```diff
@@ -854,6 +863,10 @@ bool PlayInput(const std::string& filename)
 
 	s_playMode = MODE_PLAYING;
 
+	// Wiimotes can get messed up if the game is reopened without
+	// restarting dolphin first.  Resetting the wiimotes fixes this
```

Comment:
I would prefer if you replaced this comment with a copy of the one above (because that other one is more clear):

    // Wiimotes cause desync issues if they're not reset before launching the game
    // Without a reset the emulator must be restarted to record properly

Or maybe just include the first line. Knowing that resetting Dolphin was necessary in the past isn't of much help in the future.